app/src/main/java/org/connectbot/ConsoleActivity.java

```diff
@@ -585,55 +590,49 @@ public boolean onTouch(View v, MotionEvent event) {
 					SelectionArea area = copySource.getSelectionArea();
```

Comment:
Any reason not to totally get rid of all the selectionArea/copysource stuff now?